x-pack/elastic-agent/spec/filebeat.yml

```diff
@@ -64,7 +64,7 @@ rules:
     - cloudfoundry
     - container
     - docker
-    - google-pubsub
+    - gcp-pubsub
```

Comment:
you will need to run `mage update` in the agent directory.

Reply:
Got it, thank you!